misk/web/modules/log4j/src/core/app/index.ts

```diff
@@ -0,0 +1,39 @@
+import { createApp } from 'frint';
+import { createStore } from 'frint-store';
+const BrowserRouterService = require('frint-router/BrowserRouterService'); //require has to be used until it has typings
```

Comment:
is there an issue link for this? Comments like this tend to never get cleaned up